From fed4addde0bb770555749da3bb9f67233d589d49 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@cloudera.com>
Date: Thu, 27 Sep 2012 13:18:05 -0700
Subject: [PATCH 0544/1357] CLOUDERA-BUILD. Not able to use hadoop-test.jar to run tests

  Reason: need to have junit and mockito-all JARs in the classpath for TestDFSIO to run
  Author: Alejandro Abdelnur
  Ref: CDH-8097
---
 hadoop-common-project/hadoop-common/pom.xml |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/pom.xml b/hadoop-common-project/hadoop-common/pom.xml
index 4c77ec3..de56ee2 100644
--- a/hadoop-common-project/hadoop-common/pom.xml
+++ b/hadoop-common-project/hadoop-common/pom.xml
@@ -149,7 +149,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <scope>test</scope>
+      <scope>compile</scope>
     </dependency>
     <dependency>
       <groupId>commons-lang</groupId>
@@ -184,7 +184,7 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-all</artifactId>
-      <scope>test</scope>
+      <scope>compile</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.avro</groupId>
-- 
1.7.0.4

